strategy/to_json/from_serializable.ts

```diff
@@ -5,12 +5,10 @@ import { ToJSONStrategy } from "../compose_strategy.ts";
 
 /** serialize data using `tsSerialize` on a subclass Serializable type */
 export function fromSerializable(): ToJSONStrategy {
-  return (value: Serializable | Serializable[]): JSONValue => {
+  return (value: Serializable[] | null): JSONValue => {
     if (Array.isArray(value)) {
       return value.map((item) => item.tsSerialize());
     }
-    // Objects can be null, return them without further serialization.
-    // Double cast required to return a JSONValue
-    return value ? value.tsSerialize() : value as null as JSONValue;
```

Comment:
turns out this is never reached as `toPojo(serializable)` will be recursive on `Serializable`'s

Reply:
....

Is an array of nulls possible now? Should we write a test to check that case?

Eg.

```
value = [null,null,null,null]
```

Reply:
:100: nice catch, adding the test found a bug, it's fixed now and we will support this request next release, thanks @WillForrestMindBridge